scripts/query-NPPES-resources.sh

```diff
@@ -3,10 +3,9 @@
 #update source data from endpoint source list and NPPES
 cd ../resources/prod_resources
 YEAR=$(date +%Y)
-PASTMONTH=$(date -v-1m +%B)
+PASTMONTH=$(date -v-1m +%B 2> /dev/null) || PASTMONTH=$(date -d '1 months ago' +%B)
```

Comment:
What if it is January and the PASTMONTH is December, how does this effect YEAR?

Reply:
Good catch, just fixed this so the year goes to the previous year if the previous month is December!